assets/main-search.js

```diff
@@ -0,0 +1,47 @@
+class MainSearch extends SearchForm {
+  constructor() {
+    super();
+    this.allSearchInputs = document.querySelectorAll('input[type="search"]');
+    this.setupEventListeners();
+  }
+
+  setupEventListeners () {
+    let allSearchForms = [];
+    this.allSearchInputs.forEach(input => allSearchForms.push(input.form))
+    if (allSearchForms.length > 1) {
+      allSearchForms.forEach(form =>
+        form.addEventListener('reset', this.onFormReset.bind(this))
```

Comment:
Here, I think instead of adding another `reset` listener, we could add functionality to our original function `onSearchFormReset(event){}` from the `SearchForm` class using `super` (you can see it used in a few places already in our codebase)

I think this way should work: 
```javascript
onSearchFormReset(event) {
  super.onSearchFormReset(event);
  this.keepInSync('');
}
```

Reply:
Ah right 👍 Makes sense.